ingeniamotion/configuration.py

```diff
@@ -533,9 +553,12 @@ def get_motor_pair_poles(self, servo: str = DEFAULT_SERVO, axis: int = DEFAULT_A
             Pair poles value.
 
         """
-        return self.mc.communication.get_register(
+        pair_poles = self.mc.communication.get_register(
             self.MOTOR_POLE_PAIRS_REGISTER, servo=servo, axis=axis
         )
+        if not isinstance(pair_poles, int):
+            raise IMException("Pair poles value has to be an integer")
```

Comment:
I think the message should be: 
"Received wrong value type when reading {self.MOTOR_POLE_PAIRS_REGISTER}. Got type {type(pair_poles)}, expected int type"